docs/data/material/customization/css-theme-variables/default-tokens-viewer.md

```diff
@@ -0,0 +1,63 @@
+# Default tokens viewer
```

Comment:
I'd rename the page from "Default tokens viewer" to "CSS vars viewer" or "Default CSS vars viewer" (or something similar). The former can be more easily confused with the existing ["Default theme viewer"](https://mui.com/material-ui/customization/default-theme/) page.

Also, a token is not the same as a CSS var. "token" is the concept, something we give a value to and then consume via CSS vars, sx prop, styled callbacks that access the theme, and whatever other consuming method we could have.